packages/react/src/patterns/blocks/LogoGrid/__stories__/LogoGrid.stories.js

```diff
@@ -1,43 +1,63 @@
 /**
- * Copyright IBM Corp. 2016, 2018
+ * Copyright IBM Corp. 2016, 2020
  *
  * This source code is licensed under the Apache-2.0 license found in the
  * LICENSE file in the root directory of this source tree.
  */
 
-import { object, select, text, withKnobs } from '@storybook/addon-knobs';
-import { DDS_LOGO_GRID } from '../../../../internal/FeatureFlags';
+import {
+  boolean,
+  object,
+  select,
+  text,
+  withKnobs,
+} from '@storybook/addon-knobs';
 import LogoGrid from '../LogoGrid';
 import logos from './data/logos.json';
 import React from 'react';
 import readme from '../README.md';
+import { storiesOf } from '@storybook/react';
 
-export default !DDS_LOGO_GRID
-  ? undefined
-  : {
-      title: 'Patterns (Blocks)|LogoGrid',
-      decorators: [withKnobs],
-      parameters: {
-        readme: {
-          sidebar: readme,
-        },
+storiesOf('Patterns (Blocks)|LogoGrid', module)
```

Comment:
Weighing in; My apologies for potential merge conflicts caused by #2401... That said, it'd be nice if there is a chance to migrate this change to [CSF](https://storybook.js.org/docs/formats/component-story-format/) here, too. In this way we'd probably see less diffs in this file.

Reply:
Good catch @asudoh! Yes, @reesedownes if you can convert to CSF would be great.